src/renderer/features/shared/components/library-header.tsx

```diff
@@ -234,6 +239,21 @@ export const LibraryHeaderMenu = ({
                 {onPlay && (
                     <PlayLastTextButton {...handlePlayLast.handlers} {...handlePlayLast.props} />
                 )}
+                {onArtistRadio && (
+                    <Button
+                        className={clsx(styles.wideTextButton, styles.unthemed)}
+                        classNames={{
+                            label: styles.wideTextButtonLabel,
+                            root: styles.wideTextButton,
+                        }}
+                        leftSection={<Icon icon="radio" />}
+                        onClick={onArtistRadio}
+                        size="md"
+                        variant="subtle"
+                    >
+                        {t('page.albumArtistDetail.artistRadio', { postProcess: 'sentenceCase' })}
+                    </Button>
+                )}
```

Comment:
I'm not sold on the design here.

I would:
1. remove the className and classNames as they are specifically only applied for the special text play button styles.
2. Use variant `transparent` instead